CRT.c

```diff
@@ -685,11 +689,73 @@ void CRT_restorePrivileges() {
 
 #endif /* HAVE_SETUID_ENABLED */
 
+#if defined(HAVE_MEMFD_CREATE) && !defined(NDEBUG)
+static int stderrRedirectNewFd = -1;
+static int stderrRedirectBackupFd = -1;
+#endif /* HAVE_MEMFD_CREATE && !NDEBUG */
+
+static void redirect_stderr(void) {
+#if defined(HAVE_MEMFD_CREATE) && !defined(NDEBUG)
+   stderrRedirectNewFd = memfd_create("htop.stderr-redirect", 0);
+   if (stderrRedirectNewFd < 0) {
+      // ignore failure
+      return;
+   }
+
+   stderrRedirectBackupFd = dup(STDERR_FILENO);
+   dup2(stderrRedirectNewFd, STDERR_FILENO);
+#endif /* HAVE_MEMFD_CREATE && !NDEBUG */
```

Comment:
Without `memfd_create` maybe fall back to some temp file?

Reply:
done